config/locales/server.en.yml

```diff
@@ -2520,6 +2520,8 @@ en:
       invalid_uncategorized_category_setting: 'The "Uncategorized" category cannot be selected if ''allow uncategorized topics'' is not enabled.'
       invalid_search_ranking_weights: "Value is invalid for search_ranking_weights site setting. Example: '{0.1,0.2,0.3,1.0}'. Note that maximum value for each weight is 1.0."
 
+    keywords:
```

Comment:
We could include a few examples here and add a system spec ensuring that they get used in the admin filter.

Reply:
Added some system specs and added a recently renamed site setting and used the old setting name as a keyword in case anyone searches for it.